wire-ios/Wire-iOS.xcodeproj/project.pbxproj

```diff
@@ -10452,10 +10496,15 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = F156509621DCED3000210504 /* CommonComponents-Release.xcconfig */;
 			buildSettings = {
+				"ARCHS[sdk=iphonesimulator*]" = (
+					x86_64,
+					arm64,
+				);
 				CODE_SIGN_IDENTITY = "";
 				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "Apple Development";
 				CODE_SIGN_STYLE = Manual;
 				DEVELOPMENT_TEAM = "";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator14.4]" = i386;
```

Comment:
does this still make sense ?

Reply:
I don't know, what do you think?

Reply:
I would try removing it and see if it works. Neiher the 14.4 nor the arch makes sense to me

Reply:
removed